src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState, useCallback } from 'react';
+import cn from 'classnames';
+import './Autocomplete.scss';
+import debounce from 'lodash.debounce';
+import { peopleFromServer } from '../../data/people';
+import { Person } from '../../types/Person';
+
+const MALE_SEX = 'm';
+const FEMALE_SEX = 'f';
+const DEFAULT_DELAY = 700;
+
+type Props = {
+  getSelectedUser: (person: Person) => void
+};
```

Comment:
I suggest you renaming prop to `onUserSelected` or `onUser`, etc
Per task - you need to pass debounce delay as a prop to the component